rustfmt.toml

```diff
@@ -6,3 +6,5 @@ where_single_line = true
 trailing_comma = "Vertical"
 report_fixme = "Unnumbered"
 overflow_delimited_expr = true
+comment_width = 100
```

Comment:
Although a good attempt, it looks like it needs some manual maintenance to ensure the so-called readability.